### PR TITLE
(rfc) switch to codecov.io?

### LINKED ISSUE #229: Figure out a way to merge coverage reports on Coveralls.io
Or switch to a different solution. Currently I believe a lot of coverage is being missed as the various feature tests aren't being captured.

### PR DESCRIPTION
**Pull request by bluejekyll**
fixes #229

### CODE REVIEW

File: scripts/run_kcov.sh [hunk 1]
```diff
@@ -54,12 +54,16 @@ for i in target/debug/deps/trust_dns*-* target/debug/deps/*_tests-* ; do
...
 
-if [[ "$test_count" -eq 1 ]] ; then
-  echo kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only target/kcov-* ${last_test}
-  kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only target/kcov-* ${last_test}
-elif [[ "$test_count" -gt 1 ]] ; then
-  echo kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only --merge target/kcov-*
-  kcov --coveralls-id=${TRAVIS_JOB_ID} --report-only --merge target/kcov-*
-fi
\ No newline at end of file
+# echo "----> merging and uploading to coveralls.io"
```
Comment (briansmith) [inline scripts/run_kcov.sh]: codecov.io?

Comment (bluejekyll) [reply scripts/run_kcov.sh]: Commented out all the coveralls.io stuff see the codecov.io stuff at the bottom.

**Comment by bluejekyll** [comment]
Ok, I'm liking the codecov tools a little better. If there are no objections (@briansmith?) I think I'm going to switch...

I was hoping for higher coverage numbers with their better server-side merging of reports, but that didn't actually happen.

**Comment by briansmith** [comment]
No objections from me.

**Pull request merged**